include/dlaf/sender/with_temporary_tile.h

```diff
@@ -98,7 +99,7 @@ auto withTemporaryTile(InSender&& in_sender, F&& f) {
   return std::forward<InSender>(in_sender) |
          // Start a new asynchronous scope for keeping the input tile alive
          // until all asynchronous operations are done.
-         ex::let_value(pika::unwrapping([f = std::forward<F>(f)](auto& in) mutable {
+         ex::let_value(dlaf::common::internal::Unwrapping{[f = std::forward<F>(f)](auto& in) mutable {
```

Comment:
@teonnik with this change your use case of passing a `just(ref(tile))` into `withTemporaryTile` should work (though I may be missing something...).